pkg/controller/service/service_controller.go

```diff
@@ -703,7 +708,12 @@ func (s *ServiceController) lockedUpdateLoadBalancerHosts(service *v1.Service, h
 		}
 		return nil
 	}
-
+	if err == cloudprovider.ImplementedElsewhere {
+		// ImplementedElsewhere indicates that the UpdateLoadBalancer is a nop and the
```

Comment:
Don't you need the comment above as well.

Reply:
done